src/libs/actions/Report.ts

```diff
@@ -25,70 +27,77 @@ import Visibility from '@libs/Visibility';
 import CONFIG from '@src/CONFIG';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
-import ROUTES from '@src/ROUTES';
+import ROUTES, {Route} from '@src/ROUTES';
+import {PersonalDetails, ReportActionReactions} from '@src/types/onyx';
+import {Decision, OriginalMessageIOU} from '@src/types/onyx/OriginalMessage';
+import Report, {NotificationPreference, WriteCapability} from '@src/types/onyx/Report';
+import ReportAction, {Message, ReportActionBase, ReportActions} from '@src/types/onyx/ReportAction';
+import {EmptyObject, isEmptyObject, isNotEmptyObject} from '@src/types/utils/EmptyObject';
 import * as Session from './Session';
 import * as Welcome from './Welcome';
 
-let currentUserAccountID;
+let currentUserAccountID: number;
```

Comment:
I initially implemented it like this, the problem with this approach is that it makes this file much more complicated, in multiple functions we have to check that `currentUserAccountID` is not `undefined`, when in reality all these functions are used when a user is already logged in.

I would rather add a default value here, instead of complicating the logic, please try it for yourself `gh pr checkout 28730` and let me know what do you think.

Reply:
I understand, we had this same problem with other libs like `PersonalDetails`, and the solution was to use `-1` as default value. As you can see [here](https://github.com/Expensify/App/commit/3992778b81d9c8cae8ebba0d417ea80a7a0b9c5a) `currentUserAccountID` was already default to `-1` inside the callback before migrating it to TS, so we can assume that `-1` is a safe value to use. WDYT?